dask/array/reductions.py

```diff
@@ -423,12 +428,20 @@ def min(a, axis=None, keepdims=False, split_every=None, out=None):
     )
 
 
+def chunk_min(x, axis=None, keepdims=None, *args, **kwargs):
+    if x.size == 0:
+        return np.array([sys.maxsize])
+    else:
+        r = np.min(x, axis=axis, keepdims=keepdims, *args, **kwargs)
+        return r
```

Comment:
Just a style nitpick: No need to define an intermediate `r` variable if we're only ever `return`ing it in the next line 

```suggestion
        return np.min(x, axis=axis, keepdims=keepdims, *args, **kwargs)
```